### PR TITLE
Simple Admin Mobile Nav

### LINKED ISSUE #18430: Admin Panel in mobile is not responsive
**Describe the bug**
When getting into Settings from a mobile browser, the section is not responsive at all and it even displays the top bar wrong (you need to click on the edge on a blank part to exit the settings section)

**Logs**
NA

**To Reproduce**
Set up a Metabase instance and get into it with a mobile browser or desktop browser with iphone 6/7/8 dimensions

**Expected behavior**
The whole settings section to be responsive so I can tweak settings with the browser. Also, the top bar should have an accordion so I can exit the section without guessing where the gear is)

**Screenshots**
image
mobilesettings

**Information about your Metabase Installation:**
- Your browser and the version: Brave latest
- Your operating system: Ubuntu LTS latest
- Your databases: None
- Metabase version: 40.5
- Metabase hosting environment: Docker
- Metabase internal database: H2

**Severity**
Cosmetic, but could be higher

**Additional context**
NA

### PR DESCRIPTION
**Pull request by iethree**
## Before

Currently, our admin settings page is completely un-responsive: admins cannot even see all nav items, and can't even access the exit button, leaving no way to exit admin settings except the browser back button. While we probably don't need to support full administration from mobile devices, we should at least let users exit these settings from mobile devices.

Screenshot from 2022-04-20 15-41-41

https://user-images.githubusercontent.com/30528226/164328205-4c53f7f7-a6ec-4e4a-b794-5d7165a36fc8.mp4

## Changes

- Add a very simple mobile burger nav menu for admin settings
- no changes to desktop admin settings nav
- Resolves #18430 

## After

- Admins can navigate to all settings pages
- admins can navigate away from admin settings

Screenshot from 2022-04-20 15-47-30

https://user-images.githubusercontent.com/30528226/164328474-c58855cd-c6bb-4e87-8e1f-18a37d2f7b02.mp4

### CODE REVIEW

**Review by kulyk — APPROVED** [review]
Works great! I'd wait for a review from the design team though. Added @kdoh and @mazameli to reviewers

File: frontend/src/metabase/nav/components/AdminNavbar/AdminNavbar.tsx
Comment (iethree) [reply frontend/src/metabase/nav/components/AdminNavbar/AdminNavbar.tsx:83]: yep! it's real dumb. if the nav is open, any next click closes it.

**Review by mazameli — COMMENTED** [review]
I think this is an improvement, because now you can actually navigate to pages that you just flat out couldn't before.

I just have some hesitation because now it sets you up a bit for the expectation that the pages that you're navigating to will be responsive, and that's just not the case. If we've heard user complaints that they just want to be able to get to a page at all, then I'm cool with merging this as a stop-gap. It just feels like this would more ideally be one PR in a feature epic that would more holistically address viewing the Admin Panel in mobile, you know?

**Comment by iethree** [comment]
> I think this is an improvement, because now you can actually navigate to pages that you just flat out couldn't before.
> 
> I just have some hesitation because now it sets you up a bit for the expectation that the pages that you're navigating to will be responsive, and that's just not the case. If we've heard user complaints that they just want to be able to get to a page at all, then I'm cool with merging this as a stop-gap. It just feels like this would more ideally be one PR in a feature epic that would more holistically address viewing the Admin Panel in mobile, you know?

I think the biggest issue was users couldn't even get _out_ of the admin settings when they see they're not very mobile friendly. We could simply show the exit button - but if we go down that route, it seems to make more sense to simply disable admin settings on mobile entirely. However, that would be a feature regression and would disable users from (however awkwardly) performing some admin tasks on mobile that they can do now.

This is definitely a stopgap sort of solution, but I think it's worth merging because (a) it's a definite improvement that took very little effort, and (b) it doesn't tie our hands in any way while we think of a more holistic way to deal with admin settings on mobile.

**Pull request merged**
